app/lib/pre_assembly/from_staging_location/file_set_builder.rb

```diff
@@ -27,7 +27,7 @@ def build
         when :filename, :filename_with_ocr # one resource per distinct filename (excluding extension)
           build_for_filename
         else
-          raise 'Invalid processing_configuration: must be :default or :filename'
+          raise 'Invalid processing_configuration: must be :default, :filename, or :filename_with_ocr'
         end
```

Comment:
oversight in this message we will probably never hit since it is internal use only, but there is another option not accounted for in the error message, so just making it consistent